napari_allencell_annotator/view/viewer.py

```diff
@@ -44,4 +73,79 @@ def alert(self, alert_msg: str) -> None:
         show_info(alert_msg)
 
     def get_layers(self) -> List[Layer]:
+        """
+        Returns a list of all layers in the viewer.
+        """
         return list(self.viewer.layers)
+
+    def get_all_points_layers(self) -> List[Points]:
+        """
+        Returns a list of all point layers in the viewer.
+        """
+        return [layer for layer in self.get_layers() if isinstance(layer, Points)]
+
+    def create_points_layer(self, name: str, color: str, visible: bool, ndim: int) -> Points:
+        """
+        Creates a new point layer and sets to ADD mode to allow users to select points.
+
+        Parameters
+        ----------
+        name: str
+            The name of the point layer
+        color: str
+            The face color of the points
+        visible: bool
+            Whether the point layer is visible in the viewer
+        ndim: int
+            The number of image dimensions
+
+        Returns
+        -------
+        Points
+            A new point layer
+        """
+        points_layer: Points = self.viewer.add_points(None, name=name, face_color=color, visible=visible, ndim=ndim)
+        self.set_points_layer_mode(points_layer=points_layer, mode=PointsLayerMode.ADD)
```

Comment:
`add` being default points layer doesnt make sense. Dont set mode to begin with

Reply:
removed in https://github.com/aics-int/napari-allencell-annotator/pull/90/commits/2bcd2a279c66ff37011ee8faf6844cff6323f4a8